packages/scoutgame-ui/src/components/claim/components/PointsClaimScreen/PointsClaimScreen.tsx

```diff
@@ -104,7 +178,12 @@ export function PointsClaimScreen({
               </Stack>
             </Stack>
             <Box width={{ xs: 'fit-content', md: '100%' }}>
-              <PointsClaimButton isExecuting={isExecuting} handleClaim={handleClaim} />
+              {claimData && connectedAddress !== claimData.address.toLowerCase() ? (
+                <WalletLogin />
+              ) : (
+                <PointsClaimButton isExecuting={false} handleClaim={handleWalletClaim} />
```

Comment:
this logic looks like it would show two claim buttons if i there were no onchain claim data